astropy/modeling/fitting.py

```diff
@@ -411,9 +411,7 @@ def _is_invertible(m):
         """Check if inverse of matrix can be obtained."""
         if m.shape[0] != m.shape[1]:
             return False
-        if np.linalg.matrix_rank(m) < m.shape[0]:
-            return False
-        return True
+        return np.linalg.matrix_rank(m) >= m.shape[0]
```

Comment:
This could be refactored even more aggressively:
```suggestion
        return m.shape[0] == m.shape[1] <= np.linalg.matrix_rank(m)
```
or maybe (more verbosely)
```suggestion
        return m.shape[0] == m.shape[1] and np.linalg.matrix_rank(m) >= m.shape[0]
```